src/app/Dashboard/DashboardLayoutToolbar.tsx

```diff
@@ -553,6 +572,23 @@ export const DashboardLayoutToolbar: React.FC<DashboardLayoutToolbarProps> = (_p
           downloadModal={isDownloadModal}
         />
         {deleteWarningModal}
+        {/* Confirmation modal for clearing all cards */}
+        <Modal
+          title="Confirm Clear All Cards"
+          isOpen={showClearConfirmation}
+          onClose={() => setShowClearConfirmation(false)}
+          actions={[
+            <Button key="confirm" variant="primary" onClick={handleConfirmClearAllCards}>
+              Confirm
+            </Button>,
+            <Button key="cancel" variant="link" onClick={() => setShowClearConfirmation(false)}>
+              Cancel
+            </Button>,
+          ]}
+          className="small-warning-modal"
```

Comment:
Patternfly Modal size can be adjusted using `variant` prop.

http://v4-archive.patternfly.org/v4/components/modal#small-modal

Reply:
I think we can reuse the existing modal component here:

https://github.com/cryostatio/cryostat-web/blob/main/src/app/Modal/DeleteWarningModal.tsx

If you needs the modal to be small, just define a `variant` prop to that custom modal and pass it down to the Modal.

Reply:
> I think we can reuse the existing modal component here:
> 
> https://github.com/cryostatio/cryostat-web/blob/main/src/app/Modal/DeleteWarningModal.tsx
> 
> If you needs the modal to be small, just define a `variant` prop to that custom modal and pass it down to the Modal.

Reusing the DeleteWarningModal will have the button DELETE when the warning pops up, that is why i did not re use it

Reply:
I think can define additional props to the modal component. For example,

```ts
export interface DeleteWarningProps {
  warningType: DeleteOrDisableWarningType;
  visible: boolean;
  onAccept: () => void;
  onClose: () => void;
  acceptTitle?: string;
}
```

Then, pass it down:

```tsx
<Button variant="danger" onClick={onAcceptClose}>
 {acceptTitle?? "Delete"}
</Button>
```

Reply:
U can do the same for close/cancel button if needed.

Reply:
Make sure u also add the Delete/Remove Category so that setting can be configured.